pkg/subctl/cmd/export.go

```diff
@@ -0,0 +1,102 @@
+/*
+© 2020 Red Hat, Inc. and others.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/submariner-io/lighthouse/pkg/apis/lighthouse.submariner.io/v2alpha1"
+	lighthouseClientset "github.com/submariner-io/lighthouse/pkg/client/clientset/versioned"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	exportCmd = &cobra.Command{
+		Use:   "export",
+		Short: "Exports a resource to other clusters",
+		Long:  "This command exports a resource so it is accessible to other clusters",
+	}
+	exportServiceCmd = &cobra.Command{
+		Use:   "service",
+		Short: "Exports a Service to other clusters",
+		Long:  "This command creates a ServiceExport resource with a given name which causes the Service of the same name to be accessible to other clusters",
+		Run:   exportService,
+	}
+	serviceNamespace string
+)
+
+func init() {
+	addKubeconfigFlag(exportCmd)
+	addServiceExportFlags(exportServiceCmd)
+	exportCmd.AddCommand(exportServiceCmd)
+	rootCmd.AddCommand(exportCmd)
+}
+
+func addServiceExportFlags(cmd *cobra.Command) {
+	cmd.Flags().StringVarP(&serviceNamespace, "namespace", "n", "", "Namespace of the service ot be exported")
```

Comment:
```suggestion
	cmd.Flags().StringVarP(&serviceNamespace, "namespace", "n", "", "Namespace of the service to be exported")
```

Sorry just noticed this typo 😞

Reply:
Done